src/statmodels.jl

```diff
@@ -165,12 +165,12 @@ dof_residual(obj::RegressionModel) = error("dof_residual is not defined for $(ty
 """
     params(model)
 
-Return all parameters of a model.
+Return all parameters of a model.  For a linear model, this would return coefficients as well as variance components.
```

Comment:
Wrap lines at 92 chars.

Also, couldn't we find a more general description? @dmbates gave a possible one for `coef` (and "variance components" would only apply to a *mixed* model anyway).